src/ert/config/forward_model_step.py

```diff
@@ -1,25 +1,120 @@
 import logging
-import os
-import os.path
 from dataclasses import dataclass, field
-from typing import Dict, List, Optional, Tuple, no_type_check
+from typing import (
+    Dict,
+    List,
+    Optional,
+    TypedDict,
+)
 
 from ert.substitution_list import SubstitutionList
 
-from .parse_arg_types_list import parse_arg_types_list
 from .parsing import (
-    ConfigValidationError,
-    ForwardModelStepKeys,
     SchemaItemType,
-    init_forward_model_schema,
-    lark_parse,
 )
 
 logger = logging.getLogger(__name__)
 
 
+class ForwardModelStepJSON(TypedDict):
+    """
+    A dictionary containing information about how a forward model step should be run
+    on the queue.
+
+    Attributes:
+        name: A string representing the name of the job
```

Comment:
name is obvious so maybe just omit  or at least make the description shorter.

Reply:
Removing